docs/repo-docs/reference/watch.mdx

```diff
@@ -13,9 +13,19 @@ turbo watch [tasks]
 
 ## Using `turbo watch` with persistent tasks
 
-When your script has a built-in watcher, you likely don't need to use `turbo watch`. Instead, use your script's built-in watcher and mark the task as long-running using [`"persistent": true`](/repo/docs/reference/configuration#persistent).
+Persistent tasks are marked with [`"persistent": true`](/repo/docs/reference/configuration#persistent), meaning they won't exit. Because of this, they cannot be depended on in your task graph.
 
-Persistent tasks will continue to run as usual with `turbo watch`, allowing persistent and non-persistent tasks to be run at the same time.
+This means that persistent tasks will be ignored when using `turbo watch`, working the same way they do with [`turbo run`](/repo/docs/reference/run), allowing persistent and non-persistent tasks to be run at the same time.
+
+### Dependency-aware persistent tasks
+
+When your script has a built-in watcher capable of detecting changes in dependencies, you don't need to use `turbo watch`. Instead, use your script's built-in watcher and mark the task as long-running using [`"persistent": true`](/repo/docs/reference/configuration#persistent).
```

Comment:
I would add an example of what a built-in watcher is here.

Reply:
Nice, good call.